### PR TITLE
Editing quiz app

### PR DESCRIPTION
**Pull request by alinawinter**
Please have a look

### CODE REVIEW

File: bookmarks.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    </header>
+
+    <!-- Main: Bookmarked quiz cards -->
+
+    <article class="quiz-card">
+      <a
+        class="quiz-card__bookmark--unfilled"
+        href="#bookmarks"
+        aria-label="Click to unbookmark"
+        ><i class="gg-bookmark"></i
```
Comment (alexWaligorski) [inline bookmarks.html:28]: Why do you use an i tag here? It's an empty element that is normally supposed to emphasize text.

File: bookmarks.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <footer>
+      <nav class="navbar">
+        <a
+          class="navbar__home"
+          href="./index.html"
+          aria-label="Click to go to home"
+          ><i class="gg-home-alt"></i
+        ></a>
+        <a class="navbar__bookmark--highlighted" href="./bookmarks.html"
+          ><i class="gg-bookmark"></i
```
Comment (alexWaligorski) [inline bookmarks.html:53]: Same as above. Why i tag?

File: css/global.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+.heading {
+  color: #93b0c8;
+  text-align: center;
+  font-family: "Courier New", Courier, monospace;
+  font-weight: bold;
+}
+
+.heading:hover {
```
Comment (alexWaligorski) [inline css/global.css:53]: i am not sure why the heading should change color on hover. normally this is only applyed to clickable elements. maybe remove it here so interactions are clearer?

File: css/pages/home.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  font-style: italic;
+  font-weight: bold;
+  background-color: #c2d5e5;
+}
+
+.quiz-card__answer--invisible {
+  display: none;
+}
+
+.quiz-card__tags {
```
Comment (alexWaligorski) [inline css/pages/home.css:59]: i was wonderin whether we need those tags? i thought, maybe they are only there to indicate to us students, that html, css and flexbox should be applied to this challenge :D

File: index.html [hunk 5]
```diff
@@ -16,44 +16,101 @@
...
     <footer>
       <nav class="navbar">
-        <a class="icon fa fa-home" href="#home"></a>
-        <a class="icon fa fa-bookmark-o" href="#bookmarks"></a>
-        <a class="icon fa fa-user-circle-o" href="#profile"></a>
+        <a class="navbar__home--highlighted" href="./index.html"
+          ><i class="gg-home-alt"></i
+        ></a>
+        <a
+          class="navbar__bookmark"
```
Comment (alexWaligorski) [inline index.html:103]: suggestion: unless you want to style those items differently, i wouldn't divide them into 3 classes which are indicating the content (bookmar, home...) and rather be more descriptive about the function and use one class "navbar__item" "navbar__menu-item" ....

**Review by alexWaligorski — COMMENTED** [review]
Wow! Impressive code! I learned a lot and will copy some of your ideas :D Small suggestions added.

File: index.html (hunk 5 above)
Comment (alinawinter) [reply index.html:103]: These classes are linked to icons that i imported into the file, that's why they are there. When adding additional classes, I had issues but I will try once more.

File: css/pages/home.css (hunk 4 above)
Comment (alinawinter) [reply css/pages/home.css:59]: I understood that they are supposed to be on the quizcard

File: bookmarks.html (hunk 1 above)
Comment (alinawinter) [reply bookmarks.html:28]: It is the only way that worked for me to have the bookmark-icon there that works as a link. I don't know what would be a better option here

File: css/global.css (hunk 3 above)
Comment (alinawinter) [reply css/global.css:53]: was experimenting a bit :-D you are right, I will delete it

File: bookmarks.html (hunk 1 above)
Comment (alinawinter) [reply bookmarks.html:28]: I tried removing the tag but then i cannot edit that the icon changes color, when hovering, so I left it

File: index.html (hunk 5 above)
Comment (alinawinter) [reply index.html:106]: same as above, causes issues when removing the i-tag

File: index.html [hunk 6]
```diff
@@ -16,44 +16,101 @@
...
+    <!-- Main: Quiz cards -->
+
+    <!-- Quiz card 1-->
 
     <main>
-      <article class="card one">
-        <a class="fa fa-bookmark-o" href="#bookmark_unfilled"></a>
-        <p class="question one">What property flips the axes in flexbox?</p>
-        <button type="button" class="show answer one">Show answer</button
-        ><br /><br /><br />
```
Comment (liebeckm) [inline index.html:29]: maybe work with flexbox instead of <br>? Not sure what is a better solution

File: index.html [hunk 7]
```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <link rel="stylesheet" href="./css/styles.css" />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css" />
```
Comment (liebeckm) [inline index.html:9]: I can't say if this is good or bad because I simply have no idea what you did in here 😥

File: index.html (hunk 5 above)
Comment (liebeckm) [reply index.html:106]: maybe a button could be a workaround?

**Review by liebeckm — COMMENTED** [review]
more questions than suggestions, looks nice! :)

File: index.html (hunk 6 above)
Comment (alinawinter) [reply index.html:29]: the things in red are deleted code :-D

File: index.html (hunk 7 above)
Comment (alinawinter) [reply index.html:9]: I imported icons that I used later on

File: bookmarks.html (hunk 1 above)
Comment (actyvystom) [reply bookmarks.html:28]: > Why do you use an i tag here? It's an empty element that is normally supposed to emphasize text.

The i tag is a common way to use imported icons from an icon library - looks strange, the tag is somehow "captured", but it's valid.

**Review by actyvystom — COMMENTED** [review]
Nice work! 🚀 
Looks all good to me.

**Pull request merged**